packages/webdriverio/src/commands/browser/newWindow.ts

```diff
@@ -42,7 +42,7 @@ import type { NewWindowOptions } from '../../types'
 export default async function newWindow (
     this: WebdriverIO.Browser,
     url: string,
-    { windowName = 'New Window', windowFeatures = '' }: NewWindowOptions = {}
+    { windowName = 'New Window'+Math.random()*50, windowFeatures = '' }: NewWindowOptions = {}
```

Comment:
The default values will be initialised only once, so you end up with the same random number for all calls. I suggest to remove the default window name and add it within the command if not set.

```suggestion
    { windowName = 'New Window', windowFeatures = '' }: NewWindowOptions = {}
```